hedera-node/hedera-consensus-service-impl/build.gradle.kts

```diff
@@ -28,6 +28,8 @@ configurations.all {
 }
 
 dependencies {
+  implementation(project(mapOf("path" to ":hedera-node:hedera-config")))
```

Comment:
```suggestion
  implementation(project(":hedera-node:hedera-config"))
```

Reply:
Is the `mapOf` added automatically when you let IntellIJ add the dependency or how is it created?

Reply:
IntelliJ added it. I also noticed IntelliJ added the duplicated line too.

Reply:
it ids not only about the duplicate lines. We do not want to use `mapOf`.

Reply:
Those mapOfs have been removed too.